grow/ui/sass/tool_ga_inspect.sass

```diff
@@ -0,0 +1,19 @@
+.grow__tool__ga
+  &--active
+    [data-g-event]
+      background: repeating-linear-gradient(-45deg, #FBE9E7, #FFCCBC 5px, white 5px, white 10px)
+
+      &:hover::after
+        background: #E64A19
+        border-radius: .25em
+        bottom: 50px
+        color: #ffffff
+        content: '[Event] ' attr(data-g-event) ' [Action] ' attr(data-g-action) ' [Label] ' attr(data-g-label)
```

Comment:
Hm -- I don't think we should add this to grow itself, because the `data-g-event` syntax is specific to `autotrack.js` -- and it's not even a part of airkit (yet).

Unless we want to be presumptuous about how people tag their sites, I think this needs to be configurable somehow to avoid making assumptions. That's one reason I thought it might be a good reason to make this part of airkit rather than grow, or at least start by adding support for `data-g-*` in airkit first, and then more strongly supporting the airkit+grow combination. Thoughts?